MaterialLibrary/src/main/java/io/doist/material/drawable/TintDrawable.java

```diff
@@ -150,7 +155,22 @@ private void updateTintFilter() {
     @Override
     public void draw(Canvas canvas) {
         updateTintFilter();
+
+        TintState state = mTintState;
+        final int restoreAlpha;
+        if (state.mBaseAlpha != 1.0f && state.mDrawable instanceof BitmapDrawable) {
+            final Paint p = ((BitmapDrawable) state.mDrawable).getPaint();
+            restoreAlpha = p.getAlpha();
+            p.setAlpha((int) (restoreAlpha * state.mBaseAlpha + 0.5f));
+        } else {
+            restoreAlpha = -1;
+        }
+
         super.draw(canvas);
+
+        if (restoreAlpha >= 0) {
+            ((BitmapDrawable) state.mDrawable).getPaint().setAlpha(restoreAlpha);
+        }
```

Comment:
I think the point is to always have fresh alpha of the `BitmapDrawable`, not just the `bitmap`. It's the same flow as (L's) `BitmapDrawable` uses.